packages/fluentui/react-northstar/test/utils/sharedIsConformant.tsx

```diff
@@ -0,0 +1,17 @@
+import { isConformant, IsConformantOptions } from '@fluentui/react-conformance';
+import { mountWithProvider } from 'test/utils';
+import { Ref, RefFindNode } from '@fluentui/react-component-ref';
+import { FocusZone } from '@fluentui/react-bindings';
+import * as _ from 'lodash';
+
+export function sharedIsConformant(testInfo: IsConformantOptions) {
+  const { componentPath } = testInfo;
+  const defaultConfig = {
+    customMount: mountWithProvider,
+    componentPath: componentPath.replace(/test[/\\]specs/, 'src').replace('-test.tsx', '.tsx'),
```

Comment:
This is a good idea, but I think with the current implementation the raw filename would still override the corrected one. Instead, maybe you could add a new param `testFilePath` to `sharedIsConformant`?

Reply:
Another benefit of the separate param is it's a bit misleading to be passing the test filename as **component**Path